internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,43 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/vuulpe/4th-sprint-final/internal/spentcalories"
 )
 
 var (
 	StepLength = 0.65 // длина шага в метрах
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+	parts := strings.Split(data, ",") //Spliting the string into a slice of strings.
+	if len(parts) != 2 {              //Checking the slice length is 2
+		return 0, 0, errors.New("invalid format: expected 'steps,duration'")
+	}
+
+	steps, err := strconv.Atoi(parts[0]) //converting steps number into the int type
+	if err != nil {
+		return 0, 0, fmt.Errorf("invalid steps format: %v", err) //error if wrong speps format
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err) //error if wrong speps format
```